src/publish.proj

```diff
@@ -3,13 +3,17 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <Import Project="$(ToolsDir)PublishContent.targets" />
   <Import Project="$(ToolsDir)versioning.targets" />
+  <Import Project="$(PackagesDir)/$(FeedTasksPackage)/$(FeedTasksPackageVersion)/build/$(FeedTasksPackage).targets" />
 
   <PropertyGroup>
-    <PublishPattern Condition="'$(PublishPattern)' == ''">$(PackageOutputRoot)**\*.nupkg</PublishPattern>
+    <PublishPattern Condition="'$(PublishPattern)' == ''">$(PackageOutputRoot)\**\*.nupkg</PublishPattern>
+    <SymbolsPackagesPattern>$(PackageOutputRoot)\**\*.symbols.nupkg</SymbolsPackagesPattern>
   </PropertyGroup>
 
   <ItemGroup>
     <_PackagesToPublish Include="$(PublishPattern)" />
+    <_PackagesToPublish Remove="*.*" Condition="'$(PublishSymbols)' == 'true'" />
```

Comment:
It is better to do `Remove="@(_PackagesToPublish )"` if what you are doing is trying to clear. However looking closer I'm not sure I understand what this logic is trying to accomplish.  I would expect something closer to:

```
     <_PackagesToPublish Include="$(PublishPattern)" />
     <_PackagesToPublish Remove="$(SymbolsPackagesPattern)" Condition="'$(PublishSymbols)' != 'true'" />
```